webpack/scenes/Content/Table/ContentTable.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useCallback, useEffect, useMemo } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { TableVariant, Thead, Tbody, Tr, Th, Td } from '@patternfly/react-table';
+import TableWrapper from '../../../components/Table/TableWrapper';
+import { getContent, getContentTypes } from '../ContentActions';
+import { selectContent, selectContentStatus, selectContentError, selectContentTypes, selectContentTypesStatus } from '../ContentSelectors';
+import SelectableDropdown from '../../../components/SelectableDropdown';
+import contentConfig from '../ContentConfig';
+import Loading from '../../../components/Loading';
+
+const ContentTable = () => {
+  const dispatch = useDispatch();
+  const status = useSelector(selectContentStatus);
+  const error = useSelector(selectContentError);
+  const response = useSelector(selectContent);
+  const enabledTypesResponse = useSelector(selectContentTypes);
+  const enabledTypesStatus = useSelector(selectContentTypesStatus);
+  const [contentTypeSelected, setContentTypeSelected] = useState('Python Packages');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [columnHeaders, setColumnHeaders] = useState([]);
+  const [contentTypes, setContentTypes] = useState(null);
+
+  const { results, ...metadata } = response;
+
+  function buildRows(details) {
+    const rows = [];
+    if (details) {
+      columnHeaders.forEach(header =>
+        rows.push(<Td>{header.getProperty(details)}</Td>));
```

Comment:
```suggestion
      columnHeaders.forEach((header, index) =>
        rows.push(<Td key={index}>{header.getProperty(details)}</Td>));
```
This will resolve the console error about needing unique keys. You will need to disable an eslint rule to use this.
`/* eslint-disable react/no-array-index-key */`